swagger_to_sdk_config.json

```diff
@@ -8,15 +8,17 @@
     "initScript": {
       "path": "sh scripts/automation_init.sh",
       "stderr": {
-        "showInComment": false
+        "showInComment": false,
+        "scriptError": false
```

Comment:
Is `false` allowed for `scriptError`? @wanlwanl

Reply:
yes. FYI: FYI: https://github.com/Azure/azure-rest-api-specs/blob/main/documentation/sdkautomation/SwaggerToSdkConfigSchema.json